CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- [Added logic from formatAmount UI component](https://github.com/multiversx/mx-sdk-dapp-utils/pull/7)
```

Comment:
not sure if there is a testing env prepared, but would be nice to have some specs for this function